multiversx_sdk_cli/cli_faucet.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import webbrowser
+from enum import Enum
+from typing import Any, List, Tuple
+
+from multiversx_sdk_core import Message, MessageComputer
+
+from multiversx_sdk_cli import cli_shared
+from multiversx_sdk_cli.errors import BadUserInput
+from multiversx_sdk_cli.native_auth_client import (NativeAuthClient,
+                                                   NativeAuthClientConfig)
+
+logger = logging.getLogger("cli.faucet")
+
+
+class WebWalletUrls(Enum):
+    DEVNET = "https://devnet-wallet.multiversx.com"
+    TESTNET = "https://testnet-wallet.multiversx.com"
+
+
+class ApiUrls(Enum):
+    DEVNET = "https://devnet-api.multiversx.com"
+    TESTNET = "https://testnet-api.multiversx.com"
+
+
+def setup_parser(args: List[str], subparsers: Any) -> Any:
+    parser = cli_shared.add_group_subparser(subparsers, "faucet", "Get xEGLD on Devnet or Testnet")
+    subparsers = parser.add_subparsers()
+
+    sub = cli_shared.add_command_subparser(subparsers, "faucet", "request", "Request xEGLD.")
+    cli_shared.add_wallet_args(args, sub)
+    sub.add_argument("--chain", required=True, help="the chain identifier")
```

Comment:
here it lists `--pem, --keyfile, --ledger, --chain` as required, but only one of the first three are required, right?